src/main/java/reposense/git/GitLog.java

```diff
@@ -50,4 +50,19 @@ public static String getWithFiles(RepoConfiguration config, Author author) {
 
         return runCommand(rootPath, command);
     }
+
+    /**
+     * Returns the authors who modified the binary file at {@code filePath}, in the repository specified in
+     * {@code config}.
+     */
+    public static String getBinaryFileAuthors(RepoConfiguration config, String filePath) {
+        Path rootPath = Paths.get(config.getRepoRoot());
+
+        String command = "git log --pretty=format:\"%an\t%ae\" ";
```

Comment:
I am not sure whether this is necessary - rerunning `git log` to get the list of authors that modified the file, won't that info be within the first `git log` already?

Reply:
Hmm not sure which first `git log` you mean. The only place where `git log` is used is within `generateCommitSummary`, where it returns a `CommitContributionSummary`. But `CommitContributionSummary` doesn't contain any information on authors of specific files, unless I pack the additional information required into `CommitContributionSummary`?

Reply:
The `git log` used in `CommitInfoExtractor#extractCommitInfos`

Reply:
Yeah so that method is only called in `CommitsReporter#generateCommitSummary` and returns `CommitContributionSummary` which do not contain the information about the authors of each file.